cmd/cloudflared/tunnel/cmd.go

```diff
@@ -83,12 +83,12 @@ const (
 	LogFieldTmpTraceFilename    = "tmpTraceFilename"
 	LogFieldTraceOutputFilepath = "traceOutputFilepath"
 
-	tunnelCmdErrorMessage = `You did not specify any valid additional argument to the cloudflared tunnel command. 
```

Comment:
This one is unintentional but (presumably) harmless: My editor (VS Code) strips trailing whitespaces for me.